scene/gui/file_dialog.cpp

```diff
@@ -1526,6 +1526,7 @@ FileDialog::FileDialog() {
 	update_dir();
 
 	set_hide_on_ok(false);
+	set_size(Size2(640, 360));
```

Comment:
This seems like a very brute force solution to me, and ignores things like scaling, I think the solution is elsewhere and that the size isn't computed correctly from the contents

Reply:
The size is computed correctly. It's the minimum size, which is used if you don't override it.